### PR TITLE
Replaces Surgeon General with Chief Surgeon for the CMO

### PR DESCRIPTION
**Pull request by GrayRachnid**
## About The Pull Request
Replaces Surgeon General with Chief Surgeon for the CMO

## Why It's Good For The Game
The Surgeon General is the highest ranking medical practitioner in any public office or armed service. The title is meant to be given to those who dictate how medicine is practised. This title should not be available to the senior medical practitioner of a branch.

## Changelog
:cl:
add: Added the Chief Surgeon fluff title for the CMO
del: Removed the Surgeon General fluff Title from the CMO due to its misuse.
/:cl:

### CODE REVIEW

**Comment by ariever** [comment]
Ah yes, Surgeon General Orange McFlufflypawbs its an honor

**Comment by ForFoxSake** [comment]
Not objecting, just curious. But isn't CMO _also_ the "the highest ranking medical professional within a public office"?
I thought this was a case similar to the difference between paramedic and emt, that the title differences are largely a localization difference.
Even calling them a CMO, to me, suggests that their role is primarily to dictate how medicine is practiced (alternatively, advise on matters of public health) as opposed to "organize a collection of medical professionals."

**Comment by ariever** [comment]
Chief medical officer (CMO) is the title used in many countries for the senior government official designated head of medical services, sometimes at the national level. The post is held by a physician who serves to advise and lead a team of medical experts on matters of public health importance.[1]

The post of chief medical officer dates back to Victorian times.[2] The equivalent title may go under different names across countries, for example, the Surgeon General in the United States and the Chief Public Health Officer in Canada.

*By extension, chief medical officer is also used as a job title, for the physician who is the professional lead of all physicians at a hospital.*[3]

**Comment by ForFoxSake** [comment]
Fair enough. I've also just stumbled over the discussion of this in the discord, and that's also answered my question.
Curious that I've never actually seen the term CMO used in the context of a hospital, but that's neither here nor there.
I do still feel it's worth somewhat considering that the CMO's role here _is_ centered around setting standard practice and managing health crises.
Personally I'd love to see a replacement alt title added, just so that there's still _some_ choice in what the CMO gets called. Regardless of what that title actually ends up being.

**Comment by Enzo-Leon** [comment]
I see no real reason for the removal of an alt-title like this as a Chief Medical Officer and Surgeon General have similar ties and roles in the real world. This is not to be merged unless a valid argument is provided on its removal, or a proper replacement is thought of.

**Comment by ariever** [comment]
Im not sure what you're misunderstanding but having the SURGEON GENERAL,
the person that declares how everyone in a country, army or nation should
use medicine as a playable character is a very bad idea. The person that
added Surgeon General mistakenly assumed that the term Chief Medical
Officer was pertaining to the first definition I gave, when it's being
referred to as the second in game. Surgeon General, however, is/was never
used to refer to what CMO is being used for. (Source, Wikipedia)

Secondly why is the Surgeon General, the person that dictates how medicine
should be used/preformed, working on a space station instead of, well,
being on the mainland of wherever he came from? It seems like a major
security risk to have such an important figure just bumming around on a
station in the middle of nowhere when anything could go wrong.

**Comment by Captain277** [comment]
I think you're taking this whole thing a little too seriously. No one is looking at a CMO with the alt-title Surgeon General and thinking "Oh, this is the guy that oversees all medicine in the galaxy for everyone ever". No one thought that for months, until this got brought up, anyways. It's just an optional alternate title that does nothing to change the authority or standing of the job. Everyone gets what it means. Raising this much of a fuss over it is just silly.

**Comment by GrayRachnid** [comment]
Allowing the CMO to use 'Surgeon General' as an alt-title is akin to allowing the Captain to use 'Admiral' as an alt-title. That is how big of a difference and implication the Surgeon General title gives. The Surgeon General title commands much much more authority than the title of Chief Medical Officer. The Chief Medical Officer title can and has been used in multiple different contexts ranging from small to big but the Surgeon General title has only been used in one context.

**Comment by Enzo-Leon** [comment]
We're also going with the context that the title most likely would of been adopted and changes hundreds of years into the future, and NanoTrasen taking a title as such is something not far-fetched. There would be more than one SG and they'd be overseeing operations all over in the multiple sectors that the company would manage. If we actually went with this approach we'd have to be fair and gut alt-titles from more than just the CMO.

While the SG title is a US specific role, the same power would be given to a CMO and as such we'd have to strip both in the end and give something else for them entirely. The same would need to be applied to the Chief Engineer, Security Commander, and other head of staff titles.

**Comment by GrayRachnid** [comment]
As i've explained in discord, the Surgeon General is not a US specific title/role. Second, CMO, Chief Engineer, Security Commander and any other Head of Staff title has been used in smaller contexts in the real world. Internationally and domestically for any country, the Surgeon General has only ever been given to the highest ranking Medical Practitioner that is meant to dictate how medicine is practised over whatever jurisdiction they preside over.

**Comment by ariever** [comment]
You could literally say that for every word though. And by that same logic
why do we still use PDAs when phones exist TODAY?

**Comment by Captain277** [comment]
_"Second, CMO, Chief Engineer, Security Commander and any other Head of Staff title has been used in smaller contexts in the real world."_

I'll make it really easy, watch this.
"The title Surgeon General, in the year 2564, is being used in a smaller context than it used to be."

Saying something like _"Has only ever"_ is a totally useless argument. Human space travel has only ever been viable within the inner solar system. Laser weapons have only ever been viable with giant power supplies in largely controlled situations. Running off of established precedent for something like this is utterly without merit within the context of a massive technological and cultural shift brought on by centuries of development and expansion.

**Comment by Enzo-Leon** [comment]
Good evening,

Thank you for going over the response I gave earlier, and I'm glad that we're at least on a mutual understand then on the point I'm trying to make on why the entire removal of the Surgeon General title would not really bring anything new to the table ,or fix any issues. When looking at the lore and timeline for the server and what its operations are, a Surgeon General would not be something unexpected in terms of seeing them on-board of either one of NanoTrasen's Top facilities or helping out the ships of a fleet for their mission. The setting for this server is set hundreds of years into the future, and this title could mean less of "the top medical professional that oversees the entire empire/company" to "Chief Surgeon of the sector/fleet". With lore from a similar server, they have a "Director of Health and Medicine" that would be who a SG would answer to and even THEN that position is only per sector/region for just NanoTrasen. 

When looking at a CMO/CE/HOS/PF and any of our other titles that we have for all our roles we want to ensure people are open to roleplay extra flavor into their characters, and giving them a wider job pool to come on-board and play. It also opens up more talks for players to have in-game on how they landed such a title, and what they could of done to get there as an IC promotion and medal reward is always a fun addition for the player base.

Thank you again for going over this, and I hope this clears up any concerns you had over the alt-title for the Chief Medical Officer.

- Enzo "Anthony" Leon
Citadel Station 13 Head Administrator

**Comment by ariever** [comment]
Then why not clear it up in the lore page or on the wiki?
Oh wait.

**Comment by Enzo-Leon** [comment]
Good evening @ariever,

With regards to the lore team, they've been actively working to improve and add content and information over the past few months with meetings with the playerbase and consistently receiving feedback on the matter. The is working on stuff from a high priority to lowest priority. This task in particular for an alt-title would be on the low to very-low. As I see we're not able to continue on a professional manner for this discussion we'll be looking to omitting and ignoring future commits and comments from you until an appeal is made. We apologize for any inconvenience this might cause, and thank you for your prior time on the matter. We hope to see an improvement in the future though!

**Comment by Zandario** [comment]
> Then why not clear it up in the lore page or on the wiki?
> Oh wait.

Unnecessary remark. You know very well the Lore is in the process of being rewritten. This argument is childish and none of want to deal with it. Also regarding Enzo. I appreciate the help, but that is not in your power to decide.

**Comment by GrayRachnid** [comment]
> 
> 
> Good evening,
> 
> Thank you for going over the response I gave earlier, and I'm glad that we're at least on a mutual understand then on the point I'm trying to make on why the entire removal of the Surgeon General title would not really bring anything new to the table ,or fix any issues. When looking at the lore and timeline for the server and what its operations are, a Surgeon General would not be something unexpected in terms of seeing them on-board of either one of NanoTrasen's Top facilities or helping out the ships of a fleet for their mission. The setting for this server is set hundreds of years into the future, and this title could mean less of "the top medical professional that oversees the entire empire/company" to "Chief Surgeon of the sector/fleet". With lore from a similar server, they have a "Director of Health and Medicine" that would be who a SG would answer to and even THEN that position is only per sector/region for just NanoTrasen.
> 
> When looking at a CMO/CE/HOS/PF and any of our other titles that we have for all our roles we want to ensure people are open to roleplay extra flavor into their characters, and giving them a wider job pool to come on-board and play. It also opens up more talks for players to have in-game on how they landed such a title, and what they could of done to get there as an IC promotion and medal reward is always a fun addition for the player base.
> 
> Thank you again for going over this, and I hope this clears up any concerns you had over the alt-title for the Chief Medical Officer.
> 
>     * Enzo "Anthony" Leon
>       Citadel Station 13 Head Administrator

All i'm trying to say is that the Surgeon General title has been in use since the 17th century while the Admiral title has been in used since the 13th century. Just because a couple of centuries passed doesn't change the meaning of words or their usage. Similar to how you wouldn't give the Admiral title to the Captain, you should not be giving the Surgeon General title to the CMO. In fact, it'd make even more sense to have the Captain assume the Admiral title over the CMO getting the Surgeon General title as Surgeon Generals are unique within their jurisdictions. 

I tried discussing this on discord but a maintainer told me to make a PR, so I made a PR. If you're going to deny this PR on the basis that you'll just add it to your lore, then go ahead.

Keep in mind however, there is a plethora of titles you can use to substitute this. I am not against fluff titles and giving people the opportunity to discuss how they achieved said titles, I am against giving the CMO of a ship/station/facility/colony a title that is reserved for a handful of people in our world today. Please consider changing the title to 'Head Surgeon', 'Head Physicist' or even 'Head of Medical Operations'. 

If you want to be series about your lore then please be serious about how you name titles.

**Comment by Zandario** [comment]
I'll welcome the changes when they aren't tied to insults. We're all adults here, act like it.

**Comment by Detective-Google** [comment]
> I'll welcome the changes when they aren't tied to insults. We're all adults here, act like it.

You can get started by learning to lead by example.

**Comment by GrayRachnid** [comment]
I don't know when I insulted anyone?

**Comment by Enzo-Leon** [comment]
@GrayRachnid That was not directed at you, but to the other person who was in this PR causing issues. However in your response I would actually be alright with Surgeon General going to Chief Surgeon or Head Surgeon as that would open just as much story-telling and we can reserve SG for a CCO alt-title perhaps.

**Comment by GrayRachnid** [comment]
I went with Chief Surgeon since Head Surgeon sounds like a title for someone who only does head surgeries.

File: code/game/jobs/job/medical.dm [hunk 1]
```diff
@@ -25,7 +25,7 @@
 	ideal_character_age = 50
 
 	outfit_type = /decl/hierarchy/outfit/job/medical/cmo
-	alt_titles = list ("Surgeon General","Chief Physician","Director of Medicine")
+	alt_titles = list ("Chief Physician","Director of Medicine","Chief Surgeon')
```
Comment (Zandario) [inline code/game/jobs/job/medical.dm:28]:
```suggestion
	alt_titles = list ("Chief Physician","Director of Medicine","Chief Surgeon")
```
Almost a oof

**Comment by Enzo-Leon** [comment]
@GrayRachnid Hah sounds fair for that. And yeah the discussion I had was not directed at you either mainly as someone was trying to de-rail this way further than it needed to be. Thank you for quickly adjusting it and oh yeah if you can fix that quick review note I'd be okay to +1 this.

**Comment by GrayRachnid** [comment]
done

**Comment by Zandario** [comment]
Apologies for my rash reactions, I shouldn't be doing this when I can't think straight. This is all good. +1

**Comment by kappa-sama** [comment]
@Enzo-Leon congrats on successfully applying for maintainer! I see the owner of the git still hasn't put it into effect on the github, but I have to assume you made a maintainer application and it was passed sometime recently considering your activities! Keep up the great work, new maintainer! As always, I hope to see new maintainers like yourself do some great things to increase server population and keep things running, although I understand it may be hard if you don't have too much experience in maintaining a Space Station 13 Byond Engine .dm github. If you don't have a breadth of workplace experience in the field, I do recommend that you engage in discussion with your seniors (located at the official workplace discord) before making any potentially controversial decisions.

Now that that's out of the way, we must have a discussion about your behavior thus far. As far as I can tell, you must be living in the penthouse suite of the spectrum, asshole. The only things that ever come out of your dirty, shit-lined mouth are insults degrading other people. Why is this? It's because you are a fool. An asshole. A trogolodyte. You think you are SOOO cool. You think you are SOOO gangsta. Your little weasel ass wouldn't last a day on the streets. You ask a real hood banga. You haven't lived a fraction of the stress or turmoil, yet you make that false persona your entire social appearance. Get a life.

Let us continue with the next matter. Although this may be inappropriate for a serious discussion such as this, I believe I may confide with you on a secret, as it has been nagging me for the past few days now. My chair was creaking incredibly loudly, loud enough to wake up my wife 2 rooms over. I finally had enough. I grabbed my WD-40 and sprayed around where I imagined the sound was coming from. BOOM. Easy, right? I sit down. No creak. I lean back. No Creak. "Perfect," I think to myself, "I'm a genius." As I start to sit forward-CRRRREEEEAAAAK "FUCK!" I stand up, grab the WD-40 again, and spray in 4 different places I think the sound might be coming from. I sit back down. Nothing. I lean back. Nothing. I lean forward. Nothing. "Finally, all fixed. I can do anything. I'm a literal God," I think to myself. As I start to stand up, I slide forward all the way to the edge of the seat and lean forward to stand and-CREAK I stand up. "There's only one way to fix this." I grab the chair, flip it over, grab my can of WD-40, take the precision spray straw out, and put it in tactical shotgun mode. TSSSSSSSSSSSST I'm bathing the metal support structure of my chair now, drenching it in grease. The grease runs down the underside of the metal seat frame. I pick the chair up again, flipping it upright. I spin it, and sit. Lean back. Forward. Quickly rock from side to side.. Silence.. I've done it, I've slain the creak. I truly am the most evolved being to ever exist. The Old Gods themselves would cower as I stand over them, an omni-intelligent life form who has transcended matter itself. I decide for whom the chair creaks. To celebrate my glory, I stand and get some grapes from the fridge.. But something is wrong.. My chair... Spins? Slightly faster? Effortlessly? This makes sense to me, now that I have gained all the knowledge in existence. It's obvious. In fact, it's too obvious. It's so simple, that my Ascended, Celestial, all-knowing brain failed to calculate something so simple... I've over greased the chair. And it's too late to fix. This is my life now. I'm the biggest brained man to have ever existed throughout time and space, sitting in a chair that swivels too far when I turn in it.

Essentially, with all consideration, I would say, personally, that I am most likely moderately built. However, that is just my opinion, and based on your personal definition of 'moderate' and 'built', you may simply be, built different.

Honestly, that is completely and totally fine. I would throughly enjoy a civil discussion between you and I regarding the cut of our jibs, vibes, and or physical, mental, and social 'builds'.

Personally, as I have said, I am moderately built. This means I can hold my own weight, throw my own punches, and expect results, but I am not particularly notable, and as I am aware, I am sure you are capable of feats that are, as of yet, not within my ability. Most likely, those feats will never be within my ability.

**Comment by lolman360** [comment]
make it head surgeon for the goon homage

**Pull request merged**